backend/src/main/java/com/google/sps/FindMatchQuery.java

```diff
@@ -36,23 +37,29 @@ public FindMatchQuery(Clock clock) {
   }
 
   /**
-   * Find match of new participant or add to list of participants, return whether or not match was
-   * found right after being added
+   * Find and return match of new participant with unmatched participants by comparing duration and
+   * availibility or return null if no match yet
    */
-  public Match findMatch(List<Participant> participants, Participant newParticipant) {
+  public Match findMatch(ParticipantDatastore participantDatastore, Participant newParticipant) {
```

Comment:
WDYT about making ParticipantDatastore a constructor parameter and using it as a field instead of a method arg?

Reply:
Done. Also changed in tests and still passes.